service-loadbalancer/service_loadbalancer.go

```diff
@@ -645,11 +655,11 @@ func main() {
 		glog.Fatalf("unexpected error: %v", err)
 	}
 	if !specified {
-		namespace = api.NamespaceDefault
+		namespace = api.NamespaceAll
```

Comment:
I don't recall, so services in the default namespace will get /svcName and everything else will get /ns/svcName, correct? Can you please update the readme with this (you can do so in a follow up pr if you prefer, or I'll do it when I get a chance).